lib/base/utility.cpp

```diff
@@ -1946,3 +1938,20 @@ String Utility::GetIcingaDataPath(void)
 }
 
 #endif /* _WIN32 */
+
+void Utility::RenameFile(const String& source, const String& destination)
+{
+#ifdef _WIN32
+	if (remove(destination.CStr()) && errno != ENOENT)
```

Comment:
Please explain why you've chosen remove() instead of _unlink(). Furthermore your code now throws exceptions - did you test all changed source locations and their functionality?